chia/full_node/mempool_check_conditions.py

```diff
@@ -40,38 +40,14 @@
 
 
 def get_flags_for_height_and_constants(height: int, constants: ConsensusConstants) -> int:
-    flags = 0
+    flags = ENABLE_SOFTFORK_CONDITION | ENABLE_BLS_OPS_OUTSIDE_GUARD | ENABLE_FIXED_DIV | AGG_SIG_ARGS | ALLOW_BACKREFS
```

Comment:
this is the core change. The rest are updating tests to take this into account

Reply:
This conversation is blocking merge. But it seems like an informational conversation, so resolving.